src/components/Clothe.spec.js

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react'
+import Clothe from './Clothe'
+
+describe('Clothe', () => {
+  const imageUrl = 'http://hp-api.herokuapp.com/images/harry.jpg'
+
+  it('has an image', () => {
+    render(<Clothe imageUrl={imageUrl} />)
+
+    const img = screen.getByRole('img')
+
+    expect(img).toBeInTheDocument()
+  })
+
+  it('has a alt text', () => {
```

Comment:
just a minor grammar mistake:
```suggestion
  it('has an alt text', () => {
```

You currently render your Clothing components with an empty alt text, so by definition your test isn't really what you want to test for here. You'd rather test if an "alt" as an attribute is present:
```suggestion
  it('has an alt attribute', () => {
```

Reply:
Thanks for the hint :)